src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/toric_ideal.jl

```diff
@@ -106,10 +106,9 @@ Ideal generated by
 ```
 """
 function toric_ideal(R::MPolyRing, pts::ZZMatrix)
-    intkernel = kernel(pts, side=:left)
-    presat = binomial_exponents_to_ideal(R, intkernel)
-    J = ideal([prod(gens(base_ring(presat)))])
-    return saturation(presat, J)
+    laurent, x = laurent_polynomial_ring(base_ring(R), ncols(pts); cached=false)
+    f = hom(R, laurent, [prod(x.^pts[i,:]) for i in 1:nrows(pts)])
```

Comment:
```suggestion
    f = hom(R, laurent, [laurent([one(base_ring(R))], [pts[i,:]]) for i in 1:nrows(pts)])
```
this saves some more allocations

Reply:
Nice, thanks.